HW1/P3/P3.py

```diff
@@ -0,0 +1,18 @@
+import findspark
+findspark.init()
+import pyspark
+sc = pyspark.SparkContext(appName='anagrams')
+
+with open('EOWL_words.txt', 'r') as rfile:
+	words = rfile.read().split('\n')
+
+lex_order = lambda s: ''.join(sorted(list(s)))
+
+words_rdd = sc.parallelize(words, 10)
+lex_to_word = words_rdd.map(lambda w: (lex_order(w), w))
```

Comment:
@sbalanovih This is just my personal style preference. I wouldn't have to "dig" through the code to find the lex_order() function, when it is just as reasonable to build a "long one-liner", which tells a story. 

Perhaps the best solution to this would be to name the function as explicitly as it is supposed to be used.